merino-web/src/endpoints/suggest.rs

```diff
@@ -201,6 +215,8 @@ fn safe_log_request(
         region = request.region.as_deref(),
         %query,
         client_variants = %query_params.client_variants.join(","),
+        %sid,
```

Comment:
Per this [comment](https://mozilla-hub.atlassian.net/browse/CONSVC-1839?focusedCommentId=563218), I think we can further simplify this as Merino doesn't need to provide default values for those two parameters if they're absent from the incoming requests (i.e. from older Firefox versions).

Then we could remove the above handling and just do
```rs
tracing::info!(
   ...
   sessionid = &query_params.session_id,
   sequence_no = &query_params.sequence,
   ...
)
```
It will leave out the field if it's `None`.

Reply:
Doh, just realized that `session_id` was a `Uuid` rather than `String`.

We will need to update the `tracing` crate to the latest version `0.1.35` for `merino-web`, and then do this instead:

```rs
tracing::info!(
   ...
   session_id = query_params.session_id.map(|uuid| uuid.to_string()),
   sequence_no = query_params.sequence,
   ...
)
```